pkg/storaged/utils.js

```diff
@@ -236,49 +244,69 @@ export function get_block_link_parts(client, path) {
     if (!block)
         return;
 
-    let location, link;
+    let location, link, name, subname;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18967-20230629-132526-4d4c19dd-fedora-38-devel-other//Coverage/lcov/github-pr.diff.gcov.html)